frontend/packages/operator-lifecycle-manager/locales/en/olm.json

```diff
@@ -340,6 +342,13 @@
   "0 installing": "0 installing",
   "Operator details": "Operator details",
   "Managed by": "Managed by",
+  "OperatorSource": "OperatorSource",
+  "PackageManifest": "PackageManifest",
+  "PackageManifests": "PackageManifests",
+  "ClusterServiceVersion": "ClusterServiceVersion",
+  "ClusterServiceVersions": "ClusterServiceVersions",
+  "OperatorGroups": "OperatorGroups",
+  "OperatorHubs": "OperatorHubs",
```

Comment:
Where are we using these?

Reply:
Kebab menus and breadcrumbs, mostly. I noticed one of the OLM terms wasn't internationalized on one of the OLM pages and the cause was that we didn't have i18n labelKeys on the model. I went ahead and did them all just to be safe.

Reply:
got it !